### DISCUSSION TITLE
How do I return indented json?

### DISCUSSION THREAD

**Discussion by Nebula1213** [fastapi/fastapi#6553] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [X] I commit to help with one of those options 👆

### Example Code

```python
return json.dumps({key: value}, indent=4)
```

### Description

The question is how do I indent a json response? the above code did not work.

### Operating System

Linux

### FastAPI Version

0.68.0

### Python Version

3.9.6

**Comment by Kludex** [comment, marked as answer]
You can create a custom response class: https://fastapi.tiangolo.com/advanced/custom-response/?h=custom+response+class#custom-response-class
